opt/definitions.rs

```diff
@@ -0,0 +1,114 @@
+// Fun times ahead!
+//
+// Apparently, proc-macros don't play well with `cfg_attr` yet, and their
```

Comment:
FWIW the [upstream bug](https://github.com/rust-lang/rust/issues/48644) should be closed now so once we have a new nightly this should be working again